cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperator.java

```diff
@@ -133,8 +136,8 @@ public KafkaAssemblyOperator(Vertx vertx, boolean isOpenShift,
                 .compose(state -> state.zkNodesSecret())
                 .compose(state -> state.zkNetPolicy())
                 .compose(state -> state.zkStatefulSet())
-                .compose(state -> state.zkRollingUpdate())
                 .compose(state -> state.zkScaleUp())
+                .compose(state -> state.zkRollingUpdate())
```

Comment:
Why was is necessary to swap the order of these?

Reply:
It follows the algorithm we used. It says to add pods (scale up) first and then roll the remaining pods.